_core/smart/tutorial-deploy-call.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: smart
+description: "Clarity: Deploy and Interact Tutorial"
+permalink: /:collection/:path.html
+---
+# Deploy and Interact
+
+| Experience | | **Intermediate**  |
+| Duration | | **15 minutes** |
+
+In this tutorial, you learn how to deploy and interact with a smart contract on the Stacks 2.0 blockchain. By the end of this tutorial, you will ...
+
+* Have a Clarity counter smart contract deployed to the Testnet
+* Called public methods of the deployed smart contract
+* Understand how to use the Explorer Sandbox functionality
+* Understand how to use the Blockstack CLI for advanced capabilities
+
+## Overview
+
+* TOC
+{:toc}
+
+{% include note.html content="This tutorial uses a developer preview release of the <a href='https://testnet-explorer.blockstack.org/' target='_blank'>Stacks 2.0 Explorer</a>. Please feel free to report issues or request enhancements on the <a href='https://github.com/blockstack/explorer/issues/new' target='_blank'>blockstack/explorer</a> repository." %}
+
+## Prerequisites
+
+Before you get started, you should complete the [Counter tutorial](tutorial-counter.html). Make sure your `contracts/counter.clar` file looks like this:
+
+```cl
+ (define-data-var counter int 0)
+
+ (define-public (get-counter)
+   (ok (var-get counter)))
+
+ (define-public (increment)
+   (begin
+     (var-set counter (+ (var-get counter) 1))
+     (ok (var-get counter))))
+
+ (define-public (decrement)
+   (begin
+     (var-set counter (- (var-get counter) 1))
+     (ok (var-get counter))))
+```
+
+## Step 1: Accessing the Explorer Sandbox
+
+Open up the <a href="https://testnet-explorer.blockstack.org/sandbox" target="_blank">Stacks 2.0 Explorer Sandbox view</a>. You will be asked to sign in with a Blockstack account. Follow the steps on the screen to complete the process.
```

Comment:
Should we maybe explain why a Blockstack account is needed?

Reply:
@yknl I added "so you can sign transactions when interacting with the Blockchain" -- does that cover it basically?

Reply:
I think the main reason we're using a Blockstack account here is that the key pair is not randomly re-generated every time.

Reply:
Yeah, the reason someone logs in is because the key will be handled from within the authenticator. It's not currently the case, but will be soon. The flows will be a bit different, because folks will be interacting with UI in the authenticator app.

Reply:
👍  Thanks all!